source/vscode/src/extension.ts

```diff
@@ -89,6 +90,9 @@ export async function activate(
   // fire-and-forget
   registerGhCopilotInstructionsCommand(context);
 
+  // Show prompt after update if not suppressed
+  await maybeShowChangelogPrompt(context);
```

Comment:
I'm not sure we should `await` here. This is in the activation flow, and it awaits the user response to the popup message - thus completing activation will be blocked until the uses dismisses the popup (which may trigger warnings in VS Code based on extension activation taking too long).

Reply:
removed